PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-appservice/src/main/java/com/microsoft/azure/toolkit/intellij/appservice/task/TriggerFunctionTask.java

```diff
@@ -40,7 +44,9 @@ public void execute() throws Exception {
                 .findFirst().orElse(functionEntities.get(0));
         final Action.Id<FunctionEntity> action = PlatformUtils.isIdeaUltimate() ?
                 FunctionAppActionsContributor.TRIGGER_FUNCTION_WITH_HTTP_CLIENT : FunctionAppActionsContributor.TRIGGER_FUNCTION_IN_BROWSER;
-        IntellijAzureActionManager.getInstance().getAction(action).handle(target);
+        final DataContext context = dataId -> CommonDataKeys.PROJECT.getName().equals(dataId) ? project : null;
+        final AnActionEvent event = AnActionEvent.createFromAnAction(new EmptyAction(), null, "azure.guidance.summary", context);
```

Comment:
why use `EmptyAction`, it should be `IntellijAzureActionManager.getInstance().getAction(action)`

Reply:
`IntellijAzureActionManager.getInstance().getAction` will return `Action` instead of IntelliJ `AnAction`